app/public/components/messagePanel/messages.js

```diff
@@ -19,26 +19,47 @@ import spinner from '../common/spinner.js';
 const goBack = 'Go back';
 const nothingFound = 'Nothing found';
 
-const requestButton = (model) => h('button.btn.btn-primary.m3', {
+const obsoleteRequestButton = (model) => h('button.btn.btn-primary.m3', {
```

Comment:
Meant to be removed once all the requests are done with `fetchCurrentPageData()`